packages/calling-component-bindings/src/handlers/createTeamsCallHandlers.ts

```diff
@@ -135,3 +137,16 @@ export const createTeamsCallingHandlersForComponent = <Props>(
 ): Common<TeamsCallingHandlers, Props> => {
   return createDefaultTeamsCallingHandlers(callClient, callAgent, deviceManager, call);
 };
+
+/* @conditional-compile-remove(teams-identity-support) */
+const teamsSingleParticipantTrampoline = (
+  callAgent: TeamsCallAgent,
+  participants: CommunicationIdentifier[],
+  options?: TeamsCallAgentOptions
+): TeamsCall => {
+  if (participants.length !== 1) {
+    throw new Error('Only one participant is supported in Teams call!');
+  } else {
+    return callAgent.startCall(participants[0] as any, options);
```

Comment:
is this because on the Teams CallAgent it only supports calling a teams identifier? `{teamsUserId... etc}` given how Teams currently works is it possible that the new API will be `callAgent.huddle(participants, options)` or something? just since calling multiple people on teams I think is called a `huddle` when started from a chat thread?

Reply:
Currently teams callAgent in stable only supports 1 participant - so we still keep the array api but throw errors in handlers if someone put multiple participants.